lib/src/iterable_extensions.dart

```diff
@@ -549,6 +550,54 @@ extension IterableExtension<T> on Iterable<T> {
     }
     return true;
   }
+
+  /// Returns an iterable whose elements are contiguous slices of [this].
+  ///
+  /// Each slice is [length] elements long, except for the last one which may be
+  /// shorter if [this] contains too few elements. Each slice begins after the
+  /// last one ends.
+  ///
+  /// For example, `(1, 2, 3, 4, 5).slices(2)` returns `((1, 2), (3, 4), (5))`.
+  Iterable<Iterable<T>> slices(int length) sync* {
```

Comment:
The "slices" name is not particularly telling.
Not sure which short-ish verb would better describe "chop list into equal sized chunks" through.
Ideas welcome.

Reply:
Ruby calls it `each_slice`, from which I got this name. RxJS calls it `bufferWithCount`, but that seems too strongly tied to its streaming API to use here. A Haskell package I found calls it `chunksOf`.

I suppose we could call this `chunks`, but I'm not sure that's much clearer.